src/PIL/ImageFilter.py

```diff
@@ -149,9 +149,10 @@ def filter(self, image):
 
 
 class GaussianBlur(MultibandFilter):
-    """Gaussian blur filter.
+    """Blurs the image with an extended box filter, which approximates a
+    Gaussian kernel.
```

Comment:
Is it worth to mention that approximation is very accurate?

Reply:
Instead of "very accurate" I prefer to give a notion of the accuracy. In this case it depends on a few variables so I put in the link in the C header.